src/Microsoft.TemplateEngine.Cli/CommandParsing/CommandLineHelpViewerExtensions.cs

```diff
@@ -0,0 +1,111 @@
+using Microsoft.DotNet.Cli.CommandLine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.TemplateEngine.Cli.CommandParsing
+{
+    internal static class CommandLineHelpViewerExtensions
+    {
+        private const int ColumnGutterWidth = 3;
+        private const string TemplateArgument = "TEMPLATE";
+
+        internal static string HelpView(this Command newCommand)
+        {
+            StringBuilder helpView = new StringBuilder();
+            WriteSynopsys(helpView, newCommand);
+            WriteArgumentsSection(helpView);
+            WriteOptionsSection(helpView, newCommand);
+            WriteExamples(helpView, newCommand);
+            return helpView.ToString();
+        }
+
+        private static void WriteSynopsys(StringBuilder helpView, Command command)
+        {
+            helpView.AppendLine(string.Format(LocalizableStrings.NewCommandUsage, command.Name, TemplateArgument));
```

Comment:
minor:  StringBuilder Append, return a corresponding Stringbuilder instance.   Since you're not consuming this, consider...
`_ = helpView.AppendLine(string.Format(LocalizableStrings.NewCommandUsage, command.Name, TemplateArgument));`